src/poliastro/plotting.py

```diff
@@ -480,24 +323,39 @@ def _redraw_attractor(self, min_radius=0 * u.km):
 
         # If the resulting radius is smaller than the current one, redraw it
         if radius < self._attractor_radius:
-            circle = self._plot_circle(
-                radius, BODY_COLORS.get(self._attractor.name, "#999999"), self._attractor.name)
+            if self.dim:
```

Comment:
I guess this was intended to be `dim3`? In any case, I also think this should not be here, and instead controlled in the subclass.